src/pages/are-you-sure.tsx

```diff
@@ -0,0 +1,82 @@
+import { GetServerSideProps } from "next";
+import React, { FunctionComponent } from "react";
+import { BackButton, LinkButton } from "../components/Button";
+import { Grid } from "../components/Grid";
+import { Layout } from "../components/Layout";
+import { GovUKBody, PageHeading } from "../components/Typography";
+
+interface AreYouSureProps {
+  actionText: string;
+  redirectUriIfYes: string;
+  redirectUriIfCancel: string;
+  consequencesText?: string;
+}
+
+export const AreYouSure: FunctionComponent<AreYouSureProps> = ({
+  actionText,
+  consequencesText,
+  redirectUriIfYes,
+  redirectUriIfCancel,
+}: AreYouSureProps): JSX.Element => {
+  const pageHeading = "Are you sure you want to " + actionText + "?";
+  return (
+    <Layout
+      navigation={<BackButton href={redirectUriIfCancel} />}
+      title={pageHeading}
+      showCookieBanner={false}
```

Comment:
should this be `false`? 🤔

Reply:
Initially thought the `AreYouSure` page wasn't cookie relevant because it's a bit of an in between thing, almost like a modal.  But then you asking that made me realise that's a bit of a copout, so I've fixed it :-)